devices/LEGELITE LED BULB RGBCW

```diff
@@ -0,0 +1,63 @@
+#sold on amazon. https://www.amazon.com/gp/product/B07GR4SN63/ref=ppx_yo_dt_b_asin_title_o02_s00?ie=UTF8&psc=1
+#still have some funkiness with the cold white to warm white with colors.  but for the most more it works.
+esphome:
+  name: rgb_bulb1
+  platform: ESP8266
+  board: esp01_1m
+  
+wifi:
+  ssid: '**********'
+  password: '**********'
+  manual_ip:
+    # Set this to the IP of the ESP
+    static_ip: 192.168.2.33
+    # Set this to the IP address of the router. Often ends with .1
+    gateway: 192.168.2.1
+    # The subnet of the network. 255.255.255.0 works for most home networks.
+    subnet: 255.255.255.0  
```

Comment:
Remove the manual IP section:
- it should no longer be required starting with 1.11.0
- These examples should be bare minimum and concentrate on the device itself. So the wifi/api/... config should be the default one.